### PR TITLE
addon:vela-prism

### PR DESCRIPTION
**Pull request by oeular**
Signed-off-by: Xueliang Yang <yangxueliang@cmbchina.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [ ] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: addons/vela-prism/resources/vela-prism.cue [hunk 1]
```diff
@@ -0,0 +1,14 @@
+output: {
+	type: "helm"
+	properties: {
+		repoType: "helm"
+		url:      "https://charts.kubevela.net/prism"
+		chart:    "vela-prism"
```
Comment (wangyikewxgm) [inline addons/vela-prism/resources/vela-prism.cue:6]: I think the vela-prism will rely-on the fluxcd addon if use this way. I'm not sure if this proper? ping @Somefive

**Review by diego-milhomes — APPROVED** [review]
lgtm

File: addons/vela-prism/resources/vela-prism.cue (hunk 1 above)
Comment (Somefive) [reply addons/vela-prism/resources/vela-prism.cue:6]: Let it rely on fluxcd for now. It's okay to do so.

**Review by wangyikewxgm — APPROVED** [review]
LGTM

File: addons/vela-prism/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: vela-prism
+version: v1.4.0
+description: "Prism provides API Extensions to the core KubeVela. It works as a Kubernetes Aggregated API Server."
+icon: "https://github.com/kubevela/prism/blob/master/hack/prism-arch.jpg"
```
Comment (wangyikewxgm) [inline addons/vela-prism/metadata.yaml:4]: Is it proper use the arch image as icon?

**Pull request merged**